### PR TITLE
Update pathology tumor detection readme

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes # .

### Description
This bundle takes large system RAM, thus It would be helpful to add a warning.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/pathology_tumor_detection/docs/README.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ The training pipeline is a json file (dataset.json) which includes path to each
 
 A probability number of the input patch being tumor or normal.
 
+### Memory Consumption Warning
+
+If you face memory issues in traning, you can lower the `batch_size` of the validation dataloader in the configurations to reduce the System RAM requirements.
```
Comment (KumoLiu) [inline models/pathology_tumor_detection/docs/README.md]:
```suggestion
If you face memory issues in traning, you can lower the `batch_size` or `patch_size` in the configurations to reduce the System RAM requirements.
```

Not only validation? Maybe the batch size of training could also be reduced？

File: models/pathology_tumor_detection/docs/README.md [hunk 2]
```diff
@@ -126,7 +130,7 @@ python -m monai.bundle run --config_file configs/train.json --dataset_dir <actua
 #### Override the `train` config to execute multi-GPU training
 
 ```
-torchrun --standalone --nnodes=1 --nproc_per_node=2 -m monai.bundle run --config_file "['configs/train.json','configs/multi_gpu_train.json']"
+torchrun --nnodes=1 --nproc_per_node=2 -m monai.bundle run --config_file "['configs/train.json','configs/multi_gpu_train.json']"
 ```
 
 Please note that the distributed training-related options depend on the actual running environment; thus, users may need to remove `--standalone`, modify `--nnodes`, or do some other necessary changes according to the machine used. For more details, please refer to [pytorch's official tutorial](https://pytorch.org/tutorials/intermediate/ddp_tutorial.html).
```
Comment (KumoLiu) [inline models/pathology_tumor_detection/docs/README.md]: Looks we can remove "to remove `--standalone`" here?

**Review by KumoLiu — APPROVED** [review]

File: models/pathology_tumor_detection/docs/README.md (hunk 2 above)
Comment (yiheng-wang-nv) [reply models/pathology_tumor_detection/docs/README.md]: thanks, I removed this change.

**Pull request merged**